netci.groovy

```diff
@@ -1279,18 +1299,26 @@ combinedScenarios.each { scenario ->
                                 case 'x64':
                                 case 'x86ryujit':
                                 case 'x86lb':
+                                    def arch = ''
+                                    if (architecture == 'x64') {
+                                        arch = architecture
+                                    }
+                                    else {
+                                        arch = 'x86'
+                                    }
```

Comment:
Can we just disallow x86 on non-win for now?  We can't make sure these work, so I think it's best not to even try.  We probably won't support anything but RyuJIT on Unix anyway.